src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -267,7 +267,7 @@ private async Task UseCallback_BadCertificate_ExpectedPolicyErrors_Helper(string
 
                 handler.ServerCertificateCustomValidationCallback = (request, cert, chain, errors) =>
                 {
-                    // https://github.com/dotnet/corefx/issues/21922#issuecomment-315555237
+                    // https://github.com/dotnet/runtime/issues/22644#issuecomment-315555237
```

Comment:
Personally, I dislike comments like this. If there is an important context for why the code works the way it works, it should have been extracted from the issue as a comment.